modules/visualization/dcv-eks/README.md

```diff
@@ -0,0 +1,89 @@
+
+# DCV-EKS Module
+
+## Description
+This module creates necessary resources for DCV server to work. It is 
+implemented in a fashion of one DCV server per K8S worker node (using a 
+DaemonSet). User needs to use the IP address of the node to connect to the 
+DCV  server. This module will open a NodePort for users to connect and 
+forward the traffic to DCV server. This module will also output necessary 
+information (display number and local mounting path) to configmap and aws 
+ssm parameter store. There will also be failure handling in DCV image. DCV 
+container will be restarted if the health check fails. The DCV session 
+configurations are outlined in dcv-image module.
+
+For user application pods to stream, user need to have two required inputs 
+to their pods:
+1. `DISPLAY` environment variable: usually the value is `:0`, but it is not 
+   guaranteed. User could check the actual display value in K8S ConfigMap 
+   `dcv::dcv-agent-config-map`.
```

Comment:
Can you add the relevant kubectl command to check the value?

Reply:
Yes. Adding in next update.